internal/clearingway/handlers.go

```diff
@@ -174,6 +182,47 @@ var ProgCommand = &discordgo.ApplicationCommand{
 	},
 }
 
+var ReclearCommand = &discordgo.ApplicationCommand{
+	Name:	"reclears",
+	Description: "Assign or remove reclear roles",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type: discordgo.ApplicationCommandOptionString,
+			Name: "ultimate",
+			Description: "The ultimate you want to reclear",
+			Required: true,
+			Choices: []*discordgo.ApplicationCommandOptionChoice{
+				{
+					Name: "UCoB",
+					Value: "The Unending Coil of Bahamut (Ultimate)",
+				},
+				{
+					Name: "UWU",
+					Value: "The Weapon's Refrain (Ultimate)",
+				},
+				{
+					Name: "TEA",
+					Value: "The Epic of Alexander (Ultimate)",
+				},
+				{
+					Name: "DSR",
+					Value: "Dragonsong's Reprise (Ultimate)",
+				},
+				{
+					Name: "TOP",
+					Value: "The Omega Protocol (Ultimate)",
+				},
+				/*
+				{
+					Name: "FRU",
```

Comment:
Prepared for FRU release, can be uncommented when the fight goes live.

Reply:
can we add a quick line that says `// TODO: implement when FRU goes live`